client/src/components/common/names-capture.vue

```diff
@@ -814,12 +814,10 @@ export default class NamesCapture extends NameRequestMixin {
               newReqModule.mutateNameChoices({ key: `name${choice}`, value: newName })
             }
           }
-        } else {
-          if (this.$designations[this.entity_type_cd].end && nameChoices[`designation${choice}`]) {
-            if (!nameChoices[`name${choice}`].endsWith(nameChoices[`designation${choice}`])) {
-              let newName = nameChoices[`name${choice}`] + ' ' + nameChoices[`designation${choice}`]
-              newReqModule.mutateNameChoices({ key: `name${choice}`, value: newName })
-            }
+        } else if (this.designationAtEnd && nameChoices[`designation${choice}`]) {
```

Comment:
Having now replaced the 2 conditions with the getter, it now becomes obvious that this code can never run, because it is the else case for line 810 which is `if (this.designationAtEnd)`. Anyway that can be looked at in a different scope. Today I'm just trying to fix some easy Sentry errors.

PS
```
  get designationAtEnd () {
    if (this.entity_type_cd && this.$designations[this.entity_type_cd]) {
      return this.$designations[this.entity_type_cd].end
    }
    return false
  }

```